scripts/create-substrate-fork.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+BASE_BRANCH=${2:-"moonbeam-polkadot-v0.9.12"}
+NEW_BRANCH=$1
+
+mkdir -p scripts/tmp
+cd scripts/tmp
+
+REPOS=(
+  substrate
+  polkadot
+  cumulus
+  nimbus
+  open-runtime-module-library
+  crowdloan-rewards
+  frontier
+)
+
+for REPO in ${REPOS[@]}; do
+  git clone --depth 1 git@github.com:purestake/$REPO.git -b $BASE_BRANCH
```

Comment:
Not a big deal, but it would be more efficient if we could reuse an existing repo

Reply:
(although I may be misunderstanding the point of the script)

Reply:
Each developer does not necessarily have a local copy of all the repo's and not necessarily stored in the same place. Moreover there may be diffs in the wip, and if the base branch is not local you would have to download it anyway.
In short, it will be very complex or impossible for any value, with `--depth 1` and `-b BRANCH` the download is very fast because it is reduced to the minimum.
Moreover, using a temporary local copy makes the script safer and more reproducible.